.github/workflows/release.yml

```diff
@@ -21,17 +21,27 @@ on:
         type: string
         description: Version of engflow_auth to release; must be a semver version string, starting with `v`
         required: true
+      dry_run:
+        type: boolean
```

Comment:
Warning: from my experience, boolean inputs don't work as expected in GHA: https://github.com/EngFlow/engflow/blob/dbd7172b5bf1bbce62f55ca0bab167bee42cd40c/.github/workflows/build-chromium-reclient.yml#L21